Avoid MFC dependency - can go with very basic includes instead

--- src/versioninfo.rc.in~	2015-10-16 15:40:36.000000000 +0200
+++ src/versioninfo.rc.in	2017-11-29 04:19:57.870117200 +0100
@@ -14,7 +14,6 @@
 
 #line __LINE__ "versioninfo.rc.in"
 
-#include <afxres.h>
 
 
 VS_VERSION_INFO VERSIONINFO
